### ISSUE TITLE
[Wednesday]Predict the actual quantity purchased for a given product

### ISSUE THREAD

**Issue by YuXia-SR** [YuXia-SR/dunnhumby_analysis#5]
The last step is to predict the actual quantity purchased for a given product.

- [ ] Understand the transaction data, and analyze if there’re enough price variation for a specific product to predict the demand curve
- [ ] Extract the feature to build the quantity prediction step
- [ ] Create a class object for the quantity prediction step. This class should inherit from the [BaseEstimator](https://scikit-learn.org/stable/modules/generated/sklearn.base.BaseEstimator.html) class from scikit-learn
- [ ] Fit the model and test accuracy for this step
